src/NBitcoin/IpExtensions.cs

```diff
@@ -208,20 +281,26 @@ public static bool IsMulticast(this IPAddress address)
         }
 
         /// <summary>
-        /// Specific IP address ranges that are reserved specifically as non - routable addresses to be used in 
-        /// private networks: 10.0.0.0 through 10.255.255.255. 172.16.0.0 through 172.32.255.255. 192.168.0.0 
+        /// Specific IP address ranges that are reserved specifically as non - routable addresses to be used in
+        /// private networks: 10.0.0.0 through 10.255.255.255. 172.16.0.0 through 172.32.255.255. 192.168.0.0
         /// through 192.168.255.255.
         /// </summary>
         public static bool IsRoutable(this IPAddress address, bool allowLocal)
         {
             return address.IsValid() && !(
                                             (!allowLocal && address.IsRFC1918()) ||
+                                            address.IsRFC5737() ||
+                                            address.IsRFC6598() ||
+                                            address.IsRFC7534() ||
+                                            address.IsRFC2544() ||
+                                            address.IsRFC1112() ||
```

Comment:
As RFC1112 is 240.0.0.0/4, should this be in this list? (Unless the comment is out of date)

Reply:
The reasoning is that seeing that RFC 1112 is reserved it may not be routable.

Reply:
Ah my bad @quantumagi! You are 100% correct. I didnt see the `!`.